### DISCUSSION TITLE
Swipe Back Navigation on iOS

### DISCUSSION THREAD

**Discussion by zakuru** [ionic-team/capacitor#6051] [Q&A]
Hello,

I managed to make the Swipe Back Navigation on iOS  work on iOS work by creating a new swift file and enable the feature but just curious why is that behavior not supported by default with CapacitorJS?

Thanks .

**Comment by zakuru** [comment]
Missing code was
```
import UIKit
import Capacitor
class MyViewController: CAPBridgeViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        webView!.allowsBackForwardNavigationGestures = true;
      
    }
 }
 ```

**Comment by jkjustjoshing** [comment]
This is also something that's preventing me from using Capacitor. Any answer from the team?
